app/src/main/res/values/styles.xml

```diff
@@ -4,16 +4,32 @@
     </style>
 
     <!-- Base application theme. -->
-    <style name="GrommetTheme" parent="Theme.AppCompat.Light.NoActionBar">
+    <style name="GrommetTheme" parent="Theme.MaterialComponents.Light.NoActionBar">
         <item name="android:windowDrawsSystemBarBackgrounds">true</item>
         <item name="android:statusBarColor">@android:color/transparent</item>
 
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
-        <item name="textColorError">@android:color/holo_red_dark</item>
```

Comment:
This isn't resolvable after the update. There's a comparable value in material, but all error text appears to be the proper color already regardless.

Reply:
🤷‍♂ ok, works for me